src/core/ButtonGroup/ButtonGroup.tsx

```diff
@@ -73,8 +85,15 @@ export const ButtonGroup = React.forwardRef<HTMLDivElement, ButtonGroupProps>(
       >
         {!!overflowButton && visibleCount < items.length ? (
           <>
+            {overflowButton && overflowPosition === 'start' && (
+              <div>{overflowButton(visibleCount)}</div>
+            )}
+
             {items.slice(0, visibleCount - 1)}
-            {overflowButton(visibleCount)}
+
+            {overflowButton && overflowPosition === 'end' && (
```

Comment:
I just thought that instead of `overflowButton`, we could have children as a render function so user could place  overflow button whethever they want. But I guess it will be a breaking change.